ios/ReactNativeConfig/ReactNativeConfig.m

```diff
@@ -15,6 +15,8 @@ + (NSDictionary *)env {
 }
 
 + (NSString *)envFor: (NSString *)key {
+    NSString *envValue = (NSString *)[[[NSProcessInfo processInfo]environment]objectForKey:key];
```

Comment:
Unlike android solution which captures "env" at *build time* and then exposes it via `BuildConfig` this solution for iOS is not going to work same way.

This code is executed at **runtime of the application** and not **build time**. Therefore environment variables available on machine which is used to build the app will not be captured. 

It may work when application is launched on simulator and env of osx may consumed here, it may work when launched in xcode by specifying custom runtime arguments in a scheme but it will not work when users install it from the store.